Source/ClipboardTool.spoon/init.lua

```diff
@@ -91,8 +91,8 @@ obj.show_in_menubar = true
 
 --- ClipboardTool.menubar_title
 --- Variable
---- String to show in the menubar if `ClipboardTool.show_in_menubar` is `true`. Defaults to `"\u{1f4ce}"`, which is the [Unicode paperclip character](https://codepoints.net/U+1F4CE)
-obj.menubar_title   = "\u{1f4ce}"
+--- String to show in the menubar if `ClipboardTool.show_in_menubar` is `true`. Defaults to `"\u{1f4cb}"`, which is the [Unicode paperclip character](https://codepoints.net/U+1F4CB)
```

Comment:
If we're going to change this, the word `paperclip` should also change to `clipboard`.